day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,16 @@
+# BONUS!!!!! CHANGE THE INTEGERS IN THIS ARRAY TO HAVE THE PROGRAM RUN FOR ANY RANGE OF NUMBERS!!!!
+numbers = [*1..100]
+
+numbers.each do |num|
+
+  if num % 3 == 0 && num % 5 == 0
+    puts "FizzBuzz"
+  elsif num % 5 == 0
+    puts "Buzz"
+  elsif num % 3 == 0
+    puts "Fizz"
+  else
+    puts num
+  end
+
+end
```

Comment:
Nice use of iteration, conditional statements, and modulus calculations on this!